tavern/config.go

```diff
@@ -90,7 +90,7 @@ func (cfg *Config) Connect(options ...ent.Option) (*ent.Client, error) {
 	}
 
 	var (
-		mysqlDSN = "file:ent?mode=memory&cache=shared&_fk=1"
+		mysqlDSN = "file:/tmp/test.db?cache=shared&_fk=1"
```

Comment:
I don't think we want this change going into `main`

Reply:
Yeah this was a debug moment